contrib/docker/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM python:3.6.3-alpine
+FROM python:3.6.7-alpine
```

Comment:
Why not use a more up-to-date version? And why fix it in the first place (that will probably just lead to similar security problems in the future)?

Reply:
To go higher would mean moving to 3.7, I'm not familiar enough with Python to make the call of doing that. My interest was rather the newer alpine, not python -- but the hub only had newer alpines if we upgraded python too.

Reply:
The build's not deterministic in any case - go with python:alpine